src/xdocs/config_imports.xml

```diff
@@ -607,7 +607,20 @@ import android.*;
             <td><a href="property_types.html#stringSet">String Set</a></td>
             <td><code>sun</code></td>
           </tr>
+          <tr>
+            <td>illegalClasses</td>
+            <td>class names to reject</td>
```

Comment:
please make a note in description that all properties for match will be used and describe order of matching to user.